### DISCUSSION TITLE
scATAC FindTransferAnchors: omit the first dimension?

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#543] [Q&A] [answered]
My goal is to annotate snATAC data using a reference snRNA dataset.  I am using CCA to find anchors.  When running `FindTransferAnchors()` between the ATAC gene activity matrix and pre-annotated snRNA data, should the first CCA dimension be omitted?  I am wondering because in the vignette it says there is a correlation between LSI and sequencing depth.  Does this happen with CCA too?  Looks like `FindTransferAnchors()` default is `dims = 1:30.`  Is it better to use 2:30?

**Comment by wudustan** [comment]
Hi Daniel, if your scRNA-seq is the reference then the features from the RNA will be transferred to the scATAC. The `dims` spec in `FindTransferAnchors()` would be the scRNA dims.

**Comment by timoast** [comment, marked as answer]
I don't typically find that CC1 captures sequencing depth when doing multimodal integration between scRNA and scATAC, but you could assess the correlation in your own dataset by running `RunCCA()` and then finding the correlation between total counts and each canonical component.
